sssd_test_framework/utils/sbus.py

```diff
@@ -0,0 +1,459 @@
+"""Manage D-Bus operations"""
+
+from __future__ import annotations
+
+import re
+import xml.etree.ElementTree as xml
+from abc import ABC
+from enum import Enum, auto
+from typing import Any
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.ssh import SSHProcessResult
+
+__all__ = [
+    "SBUSUtils",
+    "DBUSBus",
+    "Variant",
+]
+
+
+class DBUSBus(Enum):
+    """
+    Types of predefined D-Bus buses.
+    """
+
+    SYSTEM = auto()
+    """The system bus"""
+    SESSION = auto()
+    """The session bus"""
+
+
+class Variant:
```

Comment:
DBUSTypeVariant?

Reply:
Of course. I didn't think of renaming the type after taking it out f the `DBUSObject` class.